tests/unit/store/flask.spec.js

```diff
@@ -130,26 +130,26 @@ describe("store/flask", () => {
 
       const bound_ping_system_status = ping_system_status.bind(context);
       await bound_ping_system_status();
-
-      await store.dispatch("flask/stop_status_pinging");
+      expect(store.state.flask.status_ping_interval_id).toBe(null);
+      expect(store.state.waveform.waveform_ping_interval_id).toBe(null);
       expect(store.state.flask.status_uuid).toStrictEqual(STATUS.MESSAGE.ERROR);
     });
     test("Given all axios requests are mocked to return 404, When stop_status_pinging is dispatched, Then the system status updates to be in the ERROR state", async () => {
       mocked_axios.onGet(all_mantarray_commands_regexp).reply(404);
 
       const bound_ping_system_status = ping_system_status.bind(context);
       await bound_ping_system_status();
-
-      await store.dispatch("flask/stop_status_pinging");
+      expect(store.state.flask.status_ping_interval_id).toBe(null);
+      expect(store.state.waveform.waveform_ping_interval_id).toBe(null);
```

Comment:
`store.state.waveform.waveform_ping_interval_id` is always null at all stages of this test.  It is critical during Test Driven Development to confirm that a test fails (in the way you expect it to fail) before writing the code to make it pass.  Please adjust this test or create a new test where as part of the `Given`/`Arrange` section of the test you start the waveform interval pinging so that you can confirm that the 404 error code causes it to stop

Reply:
Let me remove the three testcases added for 300, 400 and 500 its not serving the purpose even the `status_ping_interval_id` is null irrespective of `await bound_ping_system_status()` function call.

We have to instrument the verification probably only from axios_helper.spec.js